README.Rmd

```diff
@@ -41,8 +41,8 @@ To create a reproducible research compendium using the rrtools approach, follow
 
 - this uses `devtools::create()` to create a basic R package with the name `pkgname` (you should use a different one), and then, if you're using RStudio, opens the project. If you're not using RStudio, it sets the working directory to the `pkgname` directory.
 - we need to:
-    + choose where we want our compendium package to be located on our computer. We recommend two ways you can do this. First, you can specify a full path to this function, for example, `rrtools::use_compendium("C:/Users/bmarwick/Desktop/pkgname")`. Second, you can set the working directory in RStudio using the drop-down menu: `Session` -> `Set Working Directory` and then run `rrtools::use_compendium("pkgname")`.
-    + edit the `DESCRIPTION` file to include accurate metadata
+    + choose where we want our compendium package to be located on our computer. We recommend two ways you can do this. First, you can specify a full path to this function, for example, `rrtools::use_compendium("C:/Users/bmarwick/Desktop/pkgname")`. Second, you can set the working directory by means of the setwd command, for example, `setwd("C:/Users/bmarwick/Desktop/pkgname")` or using the drop-down menu in RStudio: `Session` -> `Set Working Directory` and then run `rrtools::use_compendium("pkgname")`.
```

Comment:
Can you remove the `setwd()` from here? It is widely considered to be bad practice to use this.

Here are some sources for this opinion:

- https://stackoverflow.com/a/13770896/1036500  
- https://swcarpentry.github.io/r-novice-inflammation/06-best-practices-R/    
- http://www.cardinalpath.com/reproducibility-in-r/     
- https://books.google.com.au/books?id=n5rNBQAAQBAJ&pg=PA7&lpg=PA7&dq=setwd#v=onepage&q=setwd&f=false
- http://plantarum.ca/code/setwd/  
- https://twitter.com/hadleywickham/status/560072487753773056?lang=en
- https://gist.github.com/jennybc/362f52446fe1ebc4c49f